### PR TITLE
content: add writing a custom exporter in Go

### PR DESCRIPTION
**Pull request by odeke-em**
Updates #216

Provided a guide and runnable custom exporters for
both stats and tracing.

Changes in other languages will follow separately.

### CODE REVIEW

File: content/custom_exporter/Go/_index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+draft: false
+weight: 3
+class: "resized-logo"
+---
+
+![](/images/gopher.png)
+
+#### Introduction
+As already introduced in section [Core Concepts/Exporters](/core-concepts/exporters/), exporters
+are the vendor agnostic mechanism that OpenCensus provides to allow you to process span data
```
Comment (songy23) [inline content/custom_exporter/Go/_index.md]: "process span data" -> "process span data and metrics"

File: content/custom_exporter/Go/Metrics.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+	"go.opencensus.io/trace"
+)
+
+type customExporter struct{}
+
+// Compile time assertion that the exporter implements trace.Exporter
+var _ trace.Exporter = (*customExporter)(nil)
+
+func (cse *customExporter) ExportSpan(sd *trace.SpanData) {
```
[the remark this answers is not in the document]
Comment (odeke-em) [reply content/custom_exporter/Go/Metrics.md]: Oops, uncommitted file. Thanks for catching that @songy23. I'll push up the file I meant to commit after lunch

**Comment by odeke-em** [comment]
@songy23 I've updated with the proper file and also addressed your requested changes. PTAL.

**Comment by odeke-em** [comment]
Thank you @songy23 for the review!

**Pull request merged**